### PR TITLE
Ginkgo: Include course completion data to enrollment filter

### PR DESCRIPTION
**Pull request by amirtds**
PSU reported when the hit the batch enrollment API the data only gets filtered by enrollment creation date and not completion date of the course.

### CODE REVIEW

File: lms/djangoapps/appsembler_api/views.py [hunk 1]
```diff
@@ -718,28 +718,36 @@ def get(self, request):
         course_id = request.GET.get('course_id')
         username = request.GET.get('username')
 
-        query_filter = {}
+        enrollment_query_filter = {}
+        cert_query_filter = {}
 
         if course_id:
-            course_id = course_id.replace(' ', '+')
+            course_id= course_id.replace(' ', '+')
```
Comment (bryanlandia) [inline lms/djangoapps/appsembler_api/views.py]: Minor pep8 issue with space

Comment (amirtds) [reply lms/djangoapps/appsembler_api/views.py]: @bryanlandia  I think this takes care of the issue PSU had here https://appsembler.slack.com/archives/CEW751YQZ/p1574200602001100 wdyt ? I picked it from https://github.com/appsembler/edx-platform/commit/d00340cb6474709bfa6a38fc60792f24ee1df62f

Comment (bryanlandia) [reply lms/djangoapps/appsembler_api/views.py]: @amirtds I see... ok

**Comment by bryanlandia** [comment]
Looks good now

File: lms/djangoapps/appsembler_api/views.py (hunk 1 above)
Comment (bryanlandia) [reply lms/djangoapps/appsembler_api/views.py]: Great

**Pull request merged**